docbook/Developers_Guide/en-US/Plugins_Building.xml

```diff
@@ -90,23 +101,216 @@ class ExamplePlugin extends MantisPlugin {
+							on Mantis' manage plugin page.
+						</para>
+					</listitem>
+				</varlistentry>
+
+				<varlistentry>
+					<term>version</term>
+					<listitem>
+						<para>Your plugin's version string.
+							<emphasis>Required value.</emphasis>
+						</para>
+					</listitem>
+				</varlistentry>
+
+				<varlistentry>
+					<term>requires</term>
+					<listitem>
+						<para>An array of key/value pairs of basename/version
+							plugin dependencies.
+							<note>
+								<para>The special, reserved basename
+									<literal>MantisCore</literal> can be used to
+									specify the minimum requirement for MantisBT
+									core.
+								</para>
+							</note>
+							The version string can be defined as:
+						</para>
+						<itemizedlist>
+							<listitem>
+								<para><emphasis>Minimum requirement</emphasis>:
+									the plugin specified by the given basename
+									must be installed, and its version must be
+									equal or higher than the indicated one.
+								</para>
+							</listitem>
+							<listitem>
+								<para><emphasis>Maximum requirement</emphasis>:
+									prefixing a version number with
+									'<literal>&lt;</literal>' will allow the
+									plugin to specify the highest version
+									(non-inclusive) up to which the required
+									dependency is supported.
+									<note>
+										<para>If the plugin's minimum dependency
+											for MantisCore is unspecified or lower
+											than the current release (i.e. it does
+											not specifically list the current core
+											version as supported) and the plugin
+											does not define a maximum dependency,
+											a default one will be set to the next
+											major release of MantisBT.
+											(i.e. for 2.x.y we would add
+											<literal>'&lt;2</literal>').
+										</para>
+										<para>This effectively disables plugins
+											which have not been specifically designed
+											for a new major Mantis release, thus
+											forcing authors to review their code,
+											adapt it if necessary, and release a
+											new version of the plugin with updated
+											dependencies.
+										</para>
+									</note>
+								</para>
+							</listitem>
+							<listitem>
+								<para><emphasis>Both minimum and maximum</emphasis>:
+									the two version numbers must be separated by
+									a comma.
+								</para>
+							</listitem>
+						</itemizedlist>
+						<para>
+							Here are a few examples to illustrate the above
+							explanations, assuming that the current Mantis
+							release (<emphasis>MantisCore</emphasis> version)
+							is 2.1:
+							<itemizedlist>
+								<listitem>
+									<para>Old release without a maximum version
+										specified
+										<programlisting>
+$this->requires = array( 'MantisCore' => '1.3.1' );
+										</programlisting>
+										The plugin is compatible with MantisBT
+										>= 1.3.1 and &lt; 2(.0) - note the maximum
```

Comment:
I wrote it like that because technically the code effectively adds `<2`, and that implies the additional .0 (or even .0.0).